src/GCodePathConfig.h

```diff
@@ -29,9 +29,10 @@ class GCodePathConfig
     const int line_width; //!< width of the line extruded
     const int layer_thickness; //!< current layer height in micron
     const double flow; //!< extrusion flow modifier in %
+    const double extrusion_speed_factor; //! < extrusion speed factor (only used by purge moves for now)
```

Comment:
I don't see any reason to limit this factor to extrusion moves.

Because the factor can be set separately for any move (extrusion or travel) I think you should rename this to `speed_factor`.